### DISCUSSION TITLE
Pandoc v3.0

### DISCUSSION THREAD

**Discussion by iandol** [quarto-dev/quarto-cli#4015] [Q&A]
A pretty massive set of changes for Pandoc V3.0

https://github.com/jgm/pandoc/releases/tag/3.0

Of note for Quarto in particular are support for `mark` and a native implementation of complex figures.

There is a new chunked HTML format that could impact the book / website outputs?

This is better support for grid tables with footers for markdown input.

Lots of nice improvements, but some features overlap with Quarto's customised outputs...

**Comment by cscheid** [comment]
Yup: https://github.com/quarto-dev/quarto-cli/tree/feature/pandoc-3 :)

quarto 1.3 will ship with pandoc 3.

**Comment by iandol** [comment]
Awesome!!!
